database/seed/schema.sql

```diff
@@ -20,20 +19,20 @@ CREATE TABLE IF NOT EXISTS stock_info (
 
 CREATE TABLE IF NOT EXISTS stock_quotes (
  id SERIAL PRIMARY KEY,
- stock_id INTEGER,
+ ticker VARCHAR (5),
  price NUMERIC (7, 2),
  time_stamp TIMESTAMPTZ,
  label VARCHAR (30)
 );
 
 \copy stock_info FROM PROGRAM 'gunzip -c $(pwd)/database/dataGeneration/stockData.csv.gz' WITH DELIMITER '|' CSV HEADER;
-\copy stock_quotes(stock_id, price, time_stamp, label) FROM PROGRAM 'gunzip -c $(pwd)/database/dataGeneration/quoteData.csv.gz' WITH DELIMITER '|' CSV HEADER;
+\copy stock_quotes(ticker, price, time_stamp, label) FROM PROGRAM 'gunzip -c $(pwd)/database/dataGeneration/quoteData.csv.gz' WITH DELIMITER '|' CSV HEADER;
 
 ALTER TABLE stock_quotes
-ADD CONSTRAINT constraint_fk  FOREIGN KEY (stock_id) REFERENCES stock_info (id);
+ADD CONSTRAINT constraint_fk  FOREIGN KEY (ticker) REFERENCES stock_info (ticker);
```

Comment:
Foreign keys allow for (data validation)[https://www.bbc.com/bitesize/guides/zdvrd2p/revision/1]